packages/connections/src/lib/utils/type.ts

```diff
@@ -1,6 +1,6 @@
 type DataPropertyNames<T> = {
   // eslint-disable-next-line @typescript-eslint/ban-types
-  [K in keyof T]: T[K] extends Function ? never : K;
+  [K in keyof T]: T[K] extends Function | symbol ? never : K;
```

Comment:
`T[K]` is the value. The symbol you tried to exclude is a key. So you need to actually exclude `T extends symbol`

Reply:
Thanks for this, makes sense to me now :)